README.md

```diff
@@ -159,9 +159,9 @@ dcrpool --configfile=pool.conf --appdata=/tmp/dcrpool-harness/pool
 ```no-highlight
 rpcuser=user
 rpcpass=pass
-dcrdrpchost=127.0.0.1:19556
+dcrdrpchost=127.0.0.1:9109
 dcrdrpccert=/home/.dcrd/rpc.cert
-walletgrpchost=127.0.0.1:19558
+walletgrpchost=127.0.0.1:9111
```

Comment:
To remove a potential point of confusion, update `activenet` to `mainnet` so it matches the ports and the example above.

Reply:
Done.